src/blocking.rs

```diff
@@ -128,23 +128,15 @@ impl BlockingClient {
 
     /// Get a [`BlockHeader`] given a particular block height.
     pub fn get_header(&self, block_height: u32) -> Result<BlockHeader, Error> {
```

Comment:
Not sure if we should rename it to `get_header_by_height`.

Reply:
+1 on removing it. If anything, an API for getting a header by block_height should be implemented by Esplora, the library should just reflect what the Esplora API offers.

Reply:
I agree `get_header` should be removed since it doesn't reflect an existing Esplora API. But since we have at least one user (`bdk`) can you just [deprecate](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute) it for now?

Reply:
Deprecated with https://github.com/bitcoindevkit/rust-esplora-client/pull/17/commits/68b9d2b15366b04c28bc9dbeac1c2efb321fd63b.